src/constants/navigation.ts

```diff
@@ -1,9 +1,10 @@
 import ImageHeader from '@components/ImageHeader';
-import { WHITE } from '@constants/colors';
+import { WHITE, ALICE_BLUE } from '@constants/colors';
 
 import { HEADER_HEIGHT } from './dimensions';
 
 export const DEFAULT_NAV_OPTIONS = {
+  cardStyle: { backgroundColor: ALICE_BLUE },
```

Comment:
Why this style here???

Reply:
I used it for the backgroundColor of all screens. https://reactnavigation.org/docs/stack-navigator/#cardstyle 
What do you think?